tracing-journald/src/lib.rs

```diff
@@ -155,6 +157,31 @@ impl Subscriber {
         &self.syslog_identifier
     }
 
+    /// Sets the syslog facility for this logger.
+    ///
+    /// The syslog facility comes from the classic syslog interface (`openlog()`
+    /// and `syslog()`). In syslog the facility argument is used to specify
```

Comment:
nit:
```suggestion
    /// The syslog facility comes from the classic syslog interface (`openlog()`
    /// and `syslog()`). In syslog, the facility argument is used to specify
```

Reply:
In Finland the English language curriculum doesn't really teach how to use comma even in university level (or I was sleeping). The rule we are given is "when in doubt leave it out". 😁 Fixed.